src/WK.OpenAiWrapper/Client.storage..cs

```diff
@@ -0,0 +1,38 @@
+﻿using WK.OpenAiWrapper.Result;
+using WK.OpenAiWrapper.Models;
+using WK.OpenAiWrapper.Services;
+using OpenAI.VectorStores;
+
+namespace WK.OpenAiWrapper;
+
+internal partial class Client
+{
+    internal readonly StorageService StorageService;
+    
+    public async Task<Result<OpenAiMultipleFilesVectorStoreResponse>> UploadToNewVectorStore(string[] filePaths, string vectorStoreName, bool waitForDoneStatus = false) 
+        => await StorageService.UploadToNewVectorStore(filePaths, vectorStoreName, waitForDoneStatus).ConfigureAwait(false);
```

Comment:
For simple pass-through methods there's also the possibility to remove the "async" and "await" from the method and just return the inner Task. This can improve performance and memory consumption by a very small margin because the state machine for this async call does not have to be built and the JIT can do a simple Tail call. But it also has a drawback: the pass-though method will not be visible in stack traces, so debugging might be confusing if this fact is not known.